### ISSUE TITLE
Periodic linting of the entire file

### ISSUE THREAD

**Issue by fwcd** [fwcd/KotlinLanguageServer#86]
Lint the entire file either once the user stops typing or periodically in the background to prevent inconsistent diagnostics (see #47 and #38).

**Comment by casret** [comment]
Yeah, great idea.  I have a lot of problems with old warnings not going away.

**Comment by fwcd** [comment]
The linter is now triggered every time the document is saved. So if there are any inconsistent diagnostics, just re-save.
